ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/commitmarkable/TestCommitMarkableResourceFailAfterCommitOrphan.java

```diff
@@ -175,6 +175,8 @@ public void run() {
                     failed = true;
                 } catch (Error e) {
 
+                } finally {
+                    if(localJDBCConnection != null) try { localJDBCConnection.close(); } catch(Exception e) {};
```

Comment:
Our code style convention is to follow whatever style is present in the rest of the file. So in this case it would look like:

```java
                } finally {
                    if (localJDBCConnection != null) {
                        try {
                            localJDBCConnection.close();
                        } catch (Exception ignore) {
                        }
                    }
                }
```

BTW You should use try with resources here:

```java
                    try (Connection localJDBCConnection = dataSource.getConnection()) {
                        localJDBCConnection.setAutoCommit(false);
                        cmrResource = new JDBCConnectableResource(
                                localJDBCConnection);
                        tm.getTransaction().enlistResource(cmrResource);
                        tm.getTransaction().enlistResource(xaResource);

                        localJDBCConnection.createStatement().execute(
                                "INSERT INTO foo (bar) VALUES (1)");

                        tm.commit();
                    }
```